File: SpringMicroservices/microservice-movie/src/main/java/com/microservice/movie/infrastructure/adapters/in/rest/dto/MovieDto.java
package com.microservice.movie.infrastructure.adapters.in.rest.dto;

import lombok.*;

import java.time.LocalDate;
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class MovieDto {

    private Long idMovie;
    private String title;
    private String description;
    private Long idCategory;

    private String videoUrl480p;
    private String videoUrl720p;
    private String videoUrl1080p;

    private String audioUrlEn;
    private String audioUrlEs;

    private String subTitlesEnglish;
    private String subTitlesSpanish;

    private int durationInMinutes;
    private LocalDate releaseDate;

    private Integer seasonNumber;
    private Integer episodeNumber;
    private String imageBanner;
    private Boolean estate;
}

File: ProyFronted/app/src/main/java/com/example/proyfronted/backend/Movie/MovieDto.kt
package com.example.proyfronted.backend.Movie

data class MovieDto(
    val idMovie: Long? = null,
    val title: String,
    val description: String? = null,
    val idCategory: Long,
    val videoUrl480p: String? = null,
    val videoUrl720p: String? = null,
    val videoUrl1080p: String? = null,
    val audioUrlEn: String? = null,
    val audioUrlEs: String? = null,
    val subTitlesEnglish: String? = null,
    val subTitlesSpanish: String? = null,
    val durationInMinutes: Int,
    val releaseDate: String? = null,
    val seasonNumber: Int? = null,
    val episodeNumber: Int? = null,
    val imageBanner: String? = null,
    val estate: Boolean = true
)
